### PR TITLE
Use new logger

### PR DESCRIPTION
**Pull request by victoriazhengwf**

### CODE REVIEW

**Comment by reasonerjt** [comment]
LGTM

File: api/base.go [hunk 1]
```diff
@@ -68,17 +68,17 @@ func (b *BaseAPI) ValidateUser() int {
...
 	userID := sessionUserID.(int)
 	u, err := dao.GetUser(models.User{UserID: userID})
 	if err != nil {
-		beego.Error("Error occurred in GetUser:", err)
+		log.Errorf("Error occurred in GetUser, error: %v", err)
 		b.CustomAbort(http.StatusInternalServerError, "Internal error.")
 	}
 	if u == nil {
-		beego.Warning("User was deleted already, user id: ", userID, " canceling request.")
+		log.Warningf("User was deleted already, user id: %d ", userID, " canceling request.")
```
Comment (ywk253100) [inline api/base.go]: log.Warningf("User was deleted already, user id: %d, canceling request.", userID)

**Comment by xiaods** [comment]
LGTM

File: api/base.go [hunk 2]
```diff
@@ -17,12 +17,12 @@ package api
 
 import (
 	"encoding/json"
-	"log"
 	"net/http"
 
 	"github.com/vmware/harbor/auth"
 	"github.com/vmware/harbor/dao"
 	"github.com/vmware/harbor/models"
+	log "github.com/vmware/harbor/utils/log"
```
Comment (xiaods) [inline api/base.go]: here, we can remove alias log.

**Comment by victoriazhengwf** [comment]
fixed, and updated other files to use new log methodology in api package.

**Pull request merged**